com.ugetdm.uGet.appdata.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.ugetdm.uGet</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LGPL-2.0</project_license>
+  <name>uGet</name>
+  <summary>Download manager using GTK+ and libcurl</summary>
+  <description>
+    <p>uGet is a lightweight yet powerful Open Source download manager for GNU/Linux developed with GTK+.</p>
+    <p>Some highlights of uGet:</p>
+    <ul>
+      <li>Multi-connection downloads</li>
+      <li>Browser integration</li>
+      <li>Download YouTube videos</li>
+      <li>Clipboard monitor</li>
+      <li>Batch downloads</li>
+      <li>Downloads queue</li>
+      <li>Advanced download categories</li>
+      <li>Translated into multiple languages</li>
+      <li>Pause/resume downloads</li>
+      <li>Scheduler</li>
+      <li>After completion actions</li>
+      <li>Multiple mirrors/sources</li>
+      <li>Multiple protocols</li>
+      <li>Command-line / terminal</li>
+      <li>Quiet mode</li>
+      <li>Keyboard shortcuts</li>
+      <li>Download history</li>
+      <li>FTP login and anonymous FTP</li>
+      <li>Folder auto-creation</li>
+    </ul>
+  </description>
+  <developer_name>uGet Team</developer_name>
+  <url type="bugtracker">https://sourceforge.net/p/urlget/bugs</url>
+  <url type="donation">https://ugetdm.com/donate</url>
+  <url type="faq">https://ugetdm.com/qa/faq</url>
+  <url type="help">https://ugetdm.com/help</url>
+  <url type="homepage">https://ugetdm.com</url>
+  <url type="translate">https://ugetdm.com/translations</url>
+  <kudos>
+    <kudo>ModernToolkit</kudo>
+    <kudo>HiDpiIcon</kudo>
+    <kudo>HighContrast</kudo>
+  </kudos>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main window</caption>
+      <image>https://github.com/scx/uget-flatpak/blob/master/uget-flatpak.png?raw=true</image>
```

Comment:
Screenshot should be from upstream. If they don't have one submit it.

Reply:
I'd say they're low quality - they show too much of a user desktop

Reply:
Found one, fixed